sensor/kubernetes/enforcer/cronjob/suspend.go

```diff
@@ -0,0 +1,62 @@
+package cronjob
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/stackrox/rox/generated/internalapi/central"
+	"github.com/stackrox/rox/pkg/branding"
+	kubernetesPkg "github.com/stackrox/rox/pkg/kubernetes"
+	"github.com/stackrox/rox/pkg/retry"
+	"github.com/stackrox/rox/sensor/utils"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	batchV1      = "batch/v1"
+	batchBeta1v1 = "batch/beta1v1"
```

Comment:
You probably mean
```suggestion
	batchBeta1v1 = "batch/v1beta1"
```

Once you're ready with the change please test it on OCP 4.6 or 4.7.
Speaking of testing, would you be able to extend E2E tests for the case of CronJobs?

Reply:
Can TAL at the e2e test framework - extending that is more often than not a royal PITA.

Reply:
e2e framework has no cronjob support. maybe later, at best.

Reply:
I find absence of some code to be a questionable excuse against writing such code.
Do you mind creating a JIRA ticket for end-to-end tests for CronJob enforcement?

Reply:
Indeed, surprising the cron jobs fixes earlier didn't have tests either.
https://issues.redhat.com/browse/ROX-15452